assignment_01/app/Contracts/Services/StudentServiceInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Contracts\Services;
+
+interface StudentServiceInterface
+{
+    public function getAllStudents();
+    public function getAllMajors();
```

Comment:
If you break with empty line, then break all functions with empty line.
If your don't use new empty line, then remove new lines.
Make all the same format.